### PR TITLE
[MLOps] Refactoring image to support Vertex as a base image and parametrized kedro, spark and mlflow build

### PR DESCRIPTION
**Pull request by mwiewior**

### CODE REVIEW

File: jupyterlab-mlops/Dockerfile [hunk 1]
```diff
@@ -41,13 +72,12 @@ RUN mamba install --quiet --yes kfp jupyter-server-proxy -c conda-forge && \
 # configure python 3.8 env with kedro
 RUN mamba create --quiet --yes -p "${CONDA_DIR}/envs/python38" python=3.8 ipython ipykernel kedro=$KEDRO_VERSION && \
     mamba clean --all -f -y && \
-    "${CONDA_DIR}/envs/python38/bin/python" -m ipykernel install --user --name=python38 && \
+    "${CONDA_DIR}/envs/python38/bin/python" -m ipykernel install --user --name=kedro-python38 && \
```
Comment (szczeles) [inline jupyterlab-mlops/Dockerfile:75]: Any reason to rename the kernel?

File: jupyterlab-mlops/Dockerfile [hunk 2]
```diff
@@ -41,13 +72,12 @@ RUN mamba install --quiet --yes kfp jupyter-server-proxy -c conda-forge && \
...
     fix-permissions "/home/${NB_USER}" && \
     fix-permissions "${CONDA_DIR}"
-    
+# does above kernel installation make any sense in case of mounting home dir ?
```
Comment (szczeles) [inline jupyterlab-mlops/Dockerfile:78]: Nope, in case of home dir remount it doesn't make any sense. It works only in ephemeral KF notebooks

**Review by szczeles — APPROVED** [review]
Looks great! Consider renaming the kernel for `python38` back to `python38` to keep the backward compatibility, but it's not a must :-)

File: jupyterlab-mlops/Dockerfile (hunk 1 above)
Comment (mwiewior) [reply jupyterlab-mlops/Dockerfile]: Reverted to the old name.

File: jupyterlab-mlops/Dockerfile (hunk 2 above)
Comment (mwiewior) [reply jupyterlab-mlops/Dockerfile]: got it, comment reformulated.

**Pull request merged**